src/Core/Binary/Prims.idr

```diff
@@ -441,15 +442,18 @@ TTC Nat where
 
 ||| Get a file's modified time. If it doesn't exist, return 0 (UNIX Epoch)
```

Comment:
Can you please edit the comment to document what these Ints are?
Better yet: why not introduce a record type with meaningful field names?

Reply:
That sounds reasonable, I'll add a record time. 
I guess another improvement I should make is adding a function that would get both seconds and nanoseconds of a timestamp and return a structure from C that would be turned into the newly introduced record. This would allow us to get both parts of timestamp in a single system call and potentially prevent a dangerous situation where time is modified between two system calls.